utils.groovy

```diff
@@ -532,9 +532,22 @@ def AWSBuildUploadCredentialExists() {
     return utils.credentialsExist(creds)
 }
 
+// Grabs the jenkins.io/emoji-prefix annotation from the slack-api-token
+def getSlackEmojiPrefix() {
+    return shwrapCapture("""
+        oc get secret -n ${env.PROJECT_NAME} -o json slack-api-token | \
+          jq -r '.metadata.annotations["jenkins.io/emoji-prefix"]'
```

Comment:
If we want to handle the missing case as the caller implies, one easy way to do it is to handle it in `jq` directly here. This should work:

```
jq -r '.metadata.annotations["jenkins.io/emoji-prefix"] // ":coreos:"'
```

Reply:
To clarify, the current approach to the fallback won't work. `jq` will return "null". So a more verbose approach that would keep the fallback logic in the caller would be to check for "null" in this function and return an empty string instead.

Reply:
I think I see what you are saying. I'd prefer not to make the `jq` call any more complicated if we can.

Let's see if the latest upload addresses the concern.